src/ngraph/runtime/backend.cpp

```diff
@@ -43,7 +43,12 @@ runtime::Backend::~Backend()
 void* runtime::Backend::open_shared_library(const string& type)
 {
     void* handle = nullptr;
-    string name = "lib" + to_lower(type) + "_backend.so";
+    const string name = "lib" + to_lower(type) +
```

Comment:
Merge in the lastest master to get a better fix for this.

Reply:
Can you just pull this file from master. The code on master supports linux, macos and windows.